test/integration/branch_int_test.go

```diff
@@ -0,0 +1,65 @@
+package integration
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ActiveState/cli/internal/testhelpers/e2e"
+	"github.com/ActiveState/cli/internal/testhelpers/tagsuite"
+	"github.com/stretchr/testify/suite"
+)
+
+type BranchIntegrationTestSuite struct {
+	tagsuite.Suite
+}
+
+func (suite *BranchIntegrationTestSuite) TestBranch_List() {
+	suite.OnlyRunForTags(tagsuite.Branches)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	suite.PrepareActiveStateYAML(ts)
+
+	cp := ts.Spawn("branch")
+	expectations := []string{
+		"firstbranchchild",
+		"secondbranch",
+		"firstbranch",
+		"main",
```

Comment:
This seems badly sorted, can we sort by the date they were added or something? We should probably also show the parent/child structure, but I'll file a followup for that.

Reply:
There is no timestamp on the branch type, but I can sort it alphabetically by label

Reply:
Can we at least sort the default branch on top?